spec/lib/danger/scm_source/git_repo_spec.rb

```diff
@@ -43,7 +43,7 @@
         @dm = testing_dangerfile
         expect do
           @dm.env.scm.diff_for_folder(dir + "/subdir")
-        end.to raise_error(ArgumentError, /path does not exist/)
+        end.to raise_error(ArgumentError, /is not the top level git directory/)
```

Comment:
Previously `ArgumentError` was caused by `Git.open` called at `git_repo.rb` L20 when the path isn't a top-level git directory. But from `git` gem v1.15.0, [`Git.open` won't cause an error](https://github.com/ruby-git/ruby-git/pull/629), it doesn't cause an error any more. 

My change maintains the current intention to keep the backward compatibility but gives a better error message to educate `danger` users.

https://github.com/danger/danger/pull/1178